src/app/find-help/search-result/search-result.component.html

```diff
@@ -15,7 +15,7 @@
     <div class="right opacity-50" *ngIf="posted">{{ postedDate | date }}</div>
   </div>
   <div class="d-sm-flex">
-    <a role="button" (click)="onViewOffer()" class="flex-fill link">
+    <a role="button" class="flex-fill link">
```

Comment:
This should not be `<a>`, especially with `role="button"`, when there's no link or handler here.